api/src/types.rs

```diff
@@ -530,7 +530,7 @@ impl TxKernelPrintable {
 
 // Just the information required for wallet reconstruction
 #[derive(Debug, Serialize, Deserialize, Clone)]
-pub struct BlockHeaderInfo {
+pub struct BlockHeaderDifficultyInfo {
```

Comment:
The name of this one might be a bit confusing since it doesn't really have any difficulty info unless I'm missing something.

Reply:
The idea is just to distinguish it so it's only ever used for difficulty calculations. Since it doesn't contain any pow info and doesn't contain its own hash, I don't want anyone thinking they can rely on it for anything else